frontend/lib/src/FileUploadClient.ts

```diff
@@ -150,12 +153,24 @@ export class FileUploadClient {
    * @param fileUrl: the URL of the file to delete.
    */
   public deleteFile(fileUrl: string): Promise<void> {
-    return this.endpoints.deleteFileAtURL
```

Comment:
Does the default delete method not work correctly?

Reply:
Yes, because in stlite, all the HTTP requests to the Streamlit server have to be mocked with `kernel.sendHttpRequest()` just like `.uploadFile()`.

Reply:
oh, indeed .. I didn't think about that. The server endpoints aren't reachable from outside 👍